### DISCUSSION TITLE
Welcome to FreeDcam Discussions!

### DISCUSSION THREAD

**Discussion by KillerInk** [KillerInk/FreeDcam#137] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by massimostella** [comment]
Hi @KillerInk 
Firstly, many compliments for this amazing project and the way it is  developed already by now. 
Secondary, following what you asked for, let me introduce myself and why I'd like to participate in this community.
My name is Massimo Stella, I'm a professional filmmaker (camera operator and video editor) who's working now for some decades in the industry mostly for the main international television channels but also for other high-end productions.
Since 8 years ago, I'm part of the small core team which is developing Kdenlive, that, if you don't know what it is, is a Free and Open Source video editor application. I strongly believe in the values which sustain the development of FOSS, and I think that today smartphones can give the access to everybody to a high standard level production tool, if the hardware is combined with the right software.
So, to complete the job as advisor by bringing my experience in the field, as I'm still doing for the Kdenlive community,  I'd like to help in discussing the camcorder side of FreeDCam application.
As the majority of the applications which are around (exception for the property FilmicPro and MC24Pro), FreeDCam is mostly oriented on taking still pictures even if it is already more advanced, on the video shooting side, than other FOSS applications like Open Camera and HedgeCam2.
More than this, in the small community around the project @yzak4815, which is a professional still photographer, has already begun to give advices to improve the photo camera side, so I'd like to avoid overlapping.
Thanks again for giving the soul to this project and for aiming to create a community around it.
I'll write soon a second message with my thoughts about FreeDCAm on the "IDEAS" section, so, if you'll like, we could start our conversation from there.
Cheers.

Massimo.

**Reply by KillerInk** [reply]
you're welcome. my self is not recording movies, i only do landscape and astro photography.
thats the main reason why its more advanced for taking pics^^
Hedge and Open cam aims more for stability and use stock api.
FreeDcam work on some devices good on others not. Its depending on the OEM and how bad the api is messed up or implemented.

greets Ingo

**Comment by usyless** [comment]
Great app for video recording! 

However I do have one request that could be useful @KillerInk , would it be possible to add a manual resolution selector? Since I would like to try saving 8000x6000 DNGs or JPGs at least, (8000 6000 is listed under android.scalar.availableRawSizes in the media.camera dump of my phone).

As a side note, my phone is the oppo find x2 pro (model = CPH2025, codename = op4ba2l1)

Again, thank you for the great app and I hope you can take my idea into consideration!

**Reply by KillerInk** [reply]
Hi
FreeDcam itself support different raw sizes, if the oem did not workaround.
But on most devices with a qcfa sensor its forced to that mode and return only one raw size(binned size).

https://github.com/KillerInk/FreeDcam/blob/8dfd8d72c9c5d5130fcbf5d9f09671d235bfdcdc/app/src/main/java/freed/cam/apis/featuredetector/camera2/PictureFormatDetector.java#L46-L81

**Reply by usyless** [reply]
Oh alright, thank you for the response and yea then I guess only binned raw will be available.
